zql/docs/search-syntax/README.md

```diff
@@ -342,7 +342,7 @@ conn   1521911730.540003 CLevxl1MBUbcgovw49 10.10.18.2   57332     10.150.0.85 4
 ...
 ```
 
-Similarly, the following search will only match when the value appears in a container field of type `set[addr]` or `vector[addr]`, such as `tx_hosts` in this case.
+Similarly, the following search will only match when the value appears in a container field of type `set[addr]` or `array[addr]`, such as `tx_hosts` in this case.
```

Comment:
Not sure about this, this would be more accessible to zeek users if we use the zeek (instead of zng) terminology.
@philrz what do you think?

Reply:
Yes, it's a close call, but I'd be in favor of keeping Zeek-centric terms like `vector` in the customer-facing docs, because Zeek is the first intended use case, the sample data is Zeek, the examples speak of Zeek-centric queries, etc. @mccanne assured me that a Zeek user who sends in Zeek-format TSV will still get `vector` references in their Zeek-format TSV outputs, so it seems appropriate to still say `vector` in those contexts.